community/resource_types/airtable-baseId-tableName.json

```diff
@@ -0,0 +1,23 @@
+{
+  "workspace_id": "starter",
+  "name": "airtable-baseId-tableName",
```

Comment:
@rossmccrann would rename to airtable_table

Reply:
@rubenfiszel change made